external/dhcpd/dhcpd_lwip.c

```diff
@@ -106,24 +106,25 @@ typedef struct dhcp_join_data dhcp_join_data_s;
 void *_dhcpd_join_handler(void *arg)
 {
 	dhcp_join_data_s *data = (dhcp_join_data_s *)arg;
+	int ret = ERROR;
 	int sockfd = 0;
 	struct req_lwip_data req;
-	int ret = ERROR;
+
 	struct mq_attr attr;
 	attr.mq_maxmsg = DHCPD_MQ_MAX_LEN;
 	attr.mq_msgsize = DHCPD_MQ_LEN;
 	attr.mq_flags = 0;
 	attr.mq_curmsgs = 0;
 
-	mqd_t md = mq_open(DHCPD_MQ_NAME, O_RDWR | O_NONBLOCK |O_CREAT, 0666, &attr);
+	mqd_t md = mq_open(DHCPD_MQ_NAME, O_RDWR | O_NONBLOCK | O_CREAT, 0666, &attr);
 	if (md == (mqd_t)ERROR) {
-		ndbg("Failed to open mq\n");
+		ndbg("[DHCPD] Failed to open mq\n");
```

Comment:
Even though dbg prints function name, do you need `[DHCPD]'?
Because UART is slow, it takes much time so that we had better reduce unnecessary logs.

Reply:
That is good point, which I've never thought. All unnecessary additions are removed. Thanks for the comment.